### PR TITLE
beebeep: init at 5.8.4

### PR DESCRIPTION
**Pull request by MGlolenstine**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by MGlolenstine** [comment]
Had to fix the binary not being wrapped correctly.
The issue was in the `wrapQtAppsHook` that expected it to be at `$out/bin`.

**Comment by MGlolenstine** [comment]
I've been testing it for 3 weeks daily at work and it seems to work well.

File: pkgs/applications/office/beebeep/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  src = fetchzip {
+    url = "https://netix.dl.sourceforge.net/project/beebeep/Linux/beebeep-${version}-qt5-amd64.tar.gz";
+    sha256 = "sha256-Bi9vsPYB/jmtSLLnj2L0rOeDYz8epwNJs60kgU9kDYA=";
+  };
+
+  nativeBuildInputs = [
+    wrapQtAppsHook
+  ];
+
+  buildInputs = with pkgs; [
```
Comment (SuperSandro2000) [inline pkgs/applications/office/beebeep/default.nix:16]:
```suggestion
  buildInputs = [
```

File: pkgs/applications/office/beebeep/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{ lib, mkDerivation, pkgs, stdenv, fetchzip, wrapQtAppsHook }:
```
Comment (SuperSandro2000) [inline pkgs/applications/office/beebeep/default.nix:1]:
```suggestion
{ lib, mkDerivation, stdenv, fetchzip, wrapQtAppsHook }:
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please list the individual packages.

File: pkgs/applications/office/beebeep/default.nix (hunk 1 above)
Comment (MGlolenstine) [reply pkgs/applications/office/beebeep/default.nix:16]: If I remove both `pkgs` references, I cannot access any dependencies.

Comment (MGlolenstine) [reply pkgs/applications/office/beebeep/default.nix:16]: Did it mean I should list all of the dependencies inside the imports?

File: pkgs/applications/office/beebeep/default.nix [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  buildInputs = [
+    qtbase
+    qtmultimedia
+    qtx11extras
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    install -m755 -D beebeep $out/bin/beebeep
+    autoPatchelf $out/bin/.beebeep-wrapped
```
Comment (SuperSandro2000) [inline pkgs/applications/office/beebeep/default.nix:34]: Shouldn't this already be done by autopatchelfHook?

Comment (MGlolenstine) [reply pkgs/applications/office/beebeep/default.nix]: If I remember correctly, there were some issues with the with the `.beebeep-wrapped` not being patched.
But it seems that this isn't an issue anymore, so I'm removing it.

Removing `mkdir` and `cp`, however, doesn't build.

**Pull request merged**